frontend/packages/console-shared/src/types/resource.ts

```diff
@@ -53,3 +54,5 @@ export type OperatorBackedServiceKindMap = {
 };
 
 export type DeploymentStrategy = DEPLOYMENT_STRATEGY.recreate | DEPLOYMENT_STRATEGY.rolling;
+
+export type TopologyResources = WatchK8sResults<{ [key: string]: K8sResourceKind[] }>;
```

Comment:
This seems like it should be named more generically as there's nothing related to topology here.

Reply:
removed this